superset/assets/javascripts/explorev2/index.jsx

```diff
@@ -19,6 +19,9 @@ const bootstrappedState = Object.assign(initialState, {
   datasource_type: bootstrapData.datasource_type,
   viz: bootstrapData.viz,
 });
+bootstrappedState.viz.slice_id = parseInt(bootstrapData.slice_id, 10);
```

Comment:
could we add these keys to the viz object on the server? seems more clean than adding them here.